types/iltorb/index.d.ts

```diff
@@ -9,12 +9,15 @@
 
 import { Transform } from 'stream';
 
+export type BrotliMode = 0 | 1 | 2;
+export type BrotliCompressionQuality = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
+
 export interface BrotliEncodeParams {
     disable_literal_context_modeling?: boolean;
     lgblock?: number;
     lgwin?: number;
-    mode?: number;
-    quality?: number;
+    mode?: BrotliMode;
+    quality?: BrotliCompressionQuality;
```

Comment:
```suggestion
    /** @default 11 */
    quality?: BrotliCompressionQuality;
```
```c
#define BROTLI_DEFAULT_QUALITY 11
```

Reply:
Just realized: this may cause a bit of confusion for some users.

`iltorb` is wrapped by libraries like [`shrink-ray-current`](https://www.npmjs.com/package/shrink-ray-current#brotli). While `iltorb` is using `11` as the default value, `shrink-ray-current` is overriding that default with the value `4`. However, since the type definitions for `iltorb` as installed for you when you install `shrink-ray-current`, the hint displayed by your IDE will still claim that the default value is `11`.

I believe the solution is for `shrink-ray-current` to define its own version of `BrotliEncodeParams` with its own JSDocs, in which their default of `4` would be documented. 

(I'm leaving this comment here in case someone gets confused by the discrepancy.)